src/import-data/useImportRequest.ts

```diff
@@ -32,6 +34,9 @@ const requireString = (value: unknown, label = 'value'): string => {
   return value;
 };
 
+// A version of requireString that takes a label and returns a function that takes the value, for use in `flow`.
+const requireStringFp = (label: string) => (value: unknown) => requireString(value, label);
```

Comment:
I can back this out by not using `flow` (or change this method name) but I do prefer that style of coding.